manifests/s/solidclouds/starborne/1.3.153/solidclouds.starborne.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: solidclouds.starborne
+PackageVersion: 1.3.153
+PackageLocale: en-US
+Publisher: Solid Clouds
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: starborne
+# PackageUrl: 
+License: Solid Clouds ehf. Trademarks belong to their respective owners. All rights reserved © 2021
```

Trademarks and a copyright are not a license. Please provide the actual License
```suggestion
License: Proprietary
```